core/modules/image/tests/image.test

```diff
@@ -170,6 +170,22 @@ class ImageStylesPathAndUrlUnitTest extends BackdropWebTestCase {
   }
 
   /**
+   * Test that we do not pass an array to backdrop_add_http_header.
+   */
+  function testImageContentTypeHeaders() {
+    $files = $this->backdropGetTestFiles('image');
+    $file = array_shift($files);
+    // Copy the test file to private folder.
+    $private_uri = file_unimanaged_copy($file->uri, 'private://test.png');
+    // Tell image_module_test module to return the headers we want to test.
+    state_set('image_module_test_invalid_headers', $private_file->uri);
+    // Invoke image_style_deliver so it will try to set headers.
+    $generated_url = image_style_url($this->style_name, $private_file->uri);
```

Comment:
Thanks for updating the PR! Though now `$private_file` is missing, so `$private_file->uri` needs to be `$private_uri`.